src/components/PieChart.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="pie-chart">
+    <h3 class="pie-chart__title">
+      {{ title }}
+    </h3>
+    <div class="pie-chart__chart-container">
+      <PieWrapper
+        class="pie-chart__chart"
+        :colors="colors"
+        :labels="labels"
+        :points="points"
+        @generate-legend="(legend) => chartLegend = legend"
+      />
+      <div class="pie-chart__legend" v-html="chartLegend" />
+    </div>
+  </div>
+</template>
+
+<script>
+import PieWrapper from '@/components/ChecChart/PieWrapper.vue';
+
+export default {
+  name: 'PieChart',
+  components: {
+    PieWrapper,
+  },
+  props: {
+    title: String,
+    dataPoints: Array,
+  },
+  data() {
+    return {
+      colors: {
+        green: 'rgb(52, 202, 173)', // green-300
+        purple: 'rgb(123, 101, 205)', // purple-400
```

Comment:
Minor, but this could be a prop with a default value in case we ever want other pie graphs with more than 2 points.

Reply:
Yeah I figure that if we add more points we would use the other colours e.g. red, orange, blue, etc. We can add them as needed.